client/vite.config.js

```diff
@@ -8,5 +8,8 @@ export default defineConfig({
       '/api': 'http://localhost:4000/'
     }
   },
+  // define: {
```

Comment:
Do we need to keep this commented in or is this safe to remove/clean up?

Reply:
yup, this can be removed.